src/main/scala/firrtl/StabilizeModuleNames.scala

```diff
@@ -137,17 +137,17 @@ object StabilizeModuleNames {
     if (nameMap.values.toSet.size == modules.size) Some(nameMap) else None
   }
 
-  def appendHashCode(originalName: String, hashCode: Int): String = {
-    originalName + "_" + f"${hashCode}%08X"
+  def appendHashCode(originalName: String, hashPrefix: String, hashCode: Int): String = {
+    s"${originalName}_$hashPrefix" + f"${hashCode}%08X"
```

Comment:
do you think we really need 8 chars? Curious if others have a gut reaction that it's "long"

Reply:
I think 8 is required to fit an `Int`. I'm using the `.hashCode` method which returns `Int`.